ports/libaio/CMakeLists.txt

```
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.20)
+project(libaio
+    VERSION 0.3.113
+    DESCRIPTION "The Linux-native asynchronous I/O facility"
+)
+file(GLOB SRC_FILES src/*.c)
+add_library(aio ${SRC_FILES})
+target_include_directories(aio PUBLIC "${PROJECT_SOURCE_DIR}/src")
+install(TARGETS aio LIBRARY DESTINATION lib)
```

```suggestion
install(TARGETS aio)
```
and use the defaults. (For `x64-linux`, it is not LIBRARY but ARCHIVE.)